tasks/hivemind/agent.py

```
@@ -154,50 +156,76 @@ def detect_question_type(self) -> str:
+        rag_tool = make_rag_tool(self.enable_answer_skipping, self.community_id, self.workflow_id)
+        tools = [rag_tool]
 
-        crew = Crew(
-            agents=[q_a_bot_agent],
-            tasks=[rag_task],
-            process=Process.hierarchical,
-            manager_llm=LLM(model="gpt-4o-mini-2024-07-18"),
-            verbose=True,
+        SYSTEM_INSTRUCTIONS = f"""\
+        You are a helpful assistant.
+        """
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)